### ISSUE TITLE
Mac install - Java download failure 1.0.8 Solved

### ISSUE THREAD

**Issue by graphadvantage** [neo4j-apps/neo4j-desktop#5]
on installing on Mac latest version 1.08, hangs on java download, JDK 1.8 already installed.

```
Initialization error: Error: MD5 sum of the downloaded file did not match the expected value. Please try again!
```

other users reporting same

https://stackoverflow.com/questions/47878055/neo4j-desktop-install-error-when-downloading-java-8-on-mac

WORKAROUND:
Download http://cdn.azul.com/zulu/bin/zulu8.21.0.1-jdk8.0.131-macosx_x64.tar.gz
Copy the compressed file to:
 /Users/[USERNAME]/Library/Application Support/Neo4j Desktop/Application/distributions
Relaunch Neo4j Desktop

```

Last login: Mon Dec 18 16:19:54 on ttys001
Nicos-MacBook-Pro:~ nicomoore$ /Applications/Neo4j\ Desktop.app/Contents/MacOS/Neo4j\ Desktop ; exit;
[16:20:49:0929] [info] App starting
[16:20:50:0017] [info] Create PouchDB database if not exists - projects
[16:20:50:0018] [info] Create PouchDB database if not exists - databases
[16:20:50:0019] [info] Create PouchDB database if not exists - userData
[16:20:50:0019] [info] Create PouchDB database if not exists - graphApps
[16:20:50:0020] [info] Create PouchDB database if not exists - plugins
[16:20:50:0020] [info] Create PouchDB database if not exists - distributions
[16:20:51:0161] [info] Action[@@router/LOCATION_CHANGE] { payload: { hash: '', pathname: '/', search: '' },
  type: '@@router/LOCATION_CHANGE' }
[16:20:51:0171] [info] Function[unnamed] dispatched
[16:20:51:0171] [info] Action[APPLICATION_ONLINE] { type: 'APPLICATION_ONLINE' }
[16:20:51:0174] [info] Function[initializeIpcListenersThunk]
[16:20:51:0325] [info] Checking for update
[16:20:51:0326] [info] Checking for update.
[16:20:51:0326] [info] Action[UPDATE_CHECK] { type: 'UPDATE_CHECK' }
[16:20:51:0333] [debug] Starting migrations from index: 6
[16:20:51:0334] [info] Migrations passed: 0
[16:20:51:0339] [info] Action[APPLICATION_SHOULD_ACCEPT_AGREEMENT] { shouldAccept: false,
  type: 'APPLICATION_SHOULD_ACCEPT_AGREEMENT' }
[16:20:51:0347] [info] Action[USER_SIGNED_IN] { type: 'USER_SIGNED_IN',
  user: 
   { name: 'Nico Moore',
     picture: 'https://lh6.googleusercontent.com/-bLi02BTGWWM/AAAAAAAAAAI/AAAAAAAAABo/XjL8nHEyyrk/photo.jpg' } }
[16:20:51:0351] [info] Function[applicationInitializeThunk]
[16:20:51:0351] [info] Action[APPLICATION_INIT_START] { type: 'APPLICATION_INIT_START' }
[16:20:51:0353] [info] Java download check
[16:20:51:0354] [info] Check if java is extracted
[16:20:51:0355] [info] Action[APPLICATION_INIT_STEP_START] { step: 'JAVA_PREPARE', type: 'APPLICATION_INIT_STEP_START' }
[16:20:51:0357] [info] Action[APPLICATION_INIT_STEP_UPDATE] { payload: { name: 'Downloading Java 8' },
  step: 'JAVA_PREPARE',
  type: 'APPLICATION_INIT_STEP_UPDATE' }
[16:20:51:0363] [info] Downloading java from http://cdn.azul.com/zulu/bin/zulu8.21.0.1-jdk8.0.131-macosx_x64.tar.gz
[16:20:51:0418] [info] Download response
[16:20:51:0681] [info] Action[APPLICATION_INIT_STEP_UPDATE] { payload: 
   { progress: 
      { percent: 0.07041376713120641,
        size: [Object],
        speed: 5404657,
        time: [Object] } },
  step: 'JAVA_PREPARE',
  type: 'APPLICATION_INIT_STEP_UPDATE' }
[16:20:51:0716] [info] Update for version 1.0.8 is not available (latest version: 1.0.8, downgrade is disallowed.
[16:20:51:0716] [info] Update not available.
[16:20:51:0716] [info] Action[UPDATE_NOT_AVAILABLE] { type: 'UPDATE_NOT_AVAILABLE' }
[16:20:51:0929] [info] Action[APPLICATION_INIT_STEP_UPDATE] { payload: 
   { progress: 
      { percent: 0.16964779651633888,
        size: [Object],
        speed: 13021433,
        time: [Object] } },
  step: 'JAVA_PREPARE',
  type: 'APPLICATION_INIT_STEP_UPDATE' }
[16:20:52:0177] [info] Action[APPLICATION_INIT_STEP_UPDATE] { payload: 
   { progress: 
      { percent: 0.26619475461401,
        size: [Object],
        speed: 16345568.8,
        time: [Object] } },
  step: 'JAVA_PREPARE',
  type: 'APPLICATION_INIT_STEP_UPDATE' }
[16:20:52:0427] [info] Action[APPLICATION_INIT_STEP_UPDATE] { payload: 
   { progress: 
      { percent: 0.3611463129910662,
        size: [Object],
        speed: 18480022,
        time: [Object] } },
  step: 'JAVA_PREPARE',
  type: 'APPLICATION_INIT_STEP_UPDATE' }
[16:20:52:0675] [info] Action[APPLICATION_INIT_STEP_UPDATE] { payload: 
   { progress: 
      { percent: 0.4536384314251325,
        size: [Object],
        speed: 19896759.42857143,
        time: [Object] } },
  step: 'JAVA_PREPARE',
  type: 'APPLICATION_INIT_STEP_UPDATE' }
[16:20:52:0926] [info] Action[APPLICATION_INIT_STEP_UPDATE] { payload: 
   { progress: 
      { percent: 0.5511739807784403,
        size: [Object],
        speed: 21152868.5,
        time: [Object] } },
  step: 'JAVA_PREPARE',
  type: 'APPLICATION_INIT_STEP_UPDATE' }
[16:20:53:0175] [info] Action[APPLICATION_INIT_STEP_UPDATE] { payload: 
   { progress: 
      { percent: 0.6378211641545357,
        size: [Object],
        speed: 21758400.444444444,
        time: [Object] } },
  step: 'JAVA_PREPARE',
  type: 'APPLICATION_INIT_STEP_UPDATE' }
[16:20:53:0426] [info] Action[APPLICATION_INIT_STEP_UPDATE] { payload: 
   { progress: 
      { percent: 0.7335203414115796,
        size: [Object],
        speed: 22520742.8,
        time: [Object] } },
  step: 'JAVA_PREPARE',
  type: 'APPLICATION_INIT_STEP_UPDATE' }
[16:20:53:0675] [info] Action[APPLICATION_INIT_STEP_UPDATE] { payload: 
   { progress: 
      { percent: 0.8304144790002919,
        size: [Object],
        speed: 23177830.181818184,
        time: [Object] } },
  step: 'JAVA_PREPARE',
  type: 'APPLICATION_INIT_STEP_UPDATE' }
[16:20:53:0925] [info] Action[APPLICATION_INIT_STEP_UPDATE] { payload: 
   { progress: 
      { percent: 0.9180049149713807,
        size: [Object],
        speed: 23487365.666666668,
        time: [Object] } },
  step: 'JAVA_PREPARE',
  type: 'APPLICATION_INIT_STEP_UPDATE' }
[16:20:54:0150] [info] Download closed
[16:20:54:0156] [info] Download file stream closed
[16:20:54:0381] [error] MD5 did not match
[16:20:54:0394] [info] Action[APPLICATION_INIT_STEP_UPDATE] { payload: 
   { progress: 
      { percent: 1,
        size: [Object],
        speed: 23617134.153846152,
        time: [Object] } },
  step: 'JAVA_PREPARE',
  type: 'APPLICATION_INIT_STEP_UPDATE' }
[16:20:54:0447] [info] Action[APPLICATION_INIT_FAILED] { type: 'APPLICATION_INIT_FAILED' }


```

**Comment by oskarhane** [comment]
Thank you for the report @graphadvantage!
This is being addressed and a fix for it will be out in the next release. Sorry for the inconvenience.

**Comment by graphadvantage** [comment]
Happy to help - we really like this product.

**Comment by oskarhane** [comment]
We released 1.0.9 yesterday which fixes this issue. Feel free to re-open issue if it still exists.
